app/server/api/vendor_payout_api.py

```diff
@@ -136,62 +154,69 @@ def post(self):
             csv_data = post_data.get('csv_data', [])
             f = io.StringIO(csv_data)
             reader = csv.DictReader(f)
+
         transfers = []
         for line in reader:
-            vendor = db.session.query(TransferAccount).filter(TransferAccount.id == line['ID']).first()
-            if not vendor.primary_user.has_vendor_role:
-                response_object = {
-                    'message': str(f'{vendor} is not a vendor!'),
-                }
-                return make_response(jsonify(response_object)), 400
```

Comment:
the vendor wasn't used anywhere and the 'ID' column was confusing in the spreadsheet (is it a vendor id or a transfer id), so removed